README.md

```diff
@@ -107,21 +112,22 @@ _For instance:_
 
 `-i input.txt -o path/to/output` becomes `path/to/output/input_capice.txt.gz`
 
+- -c / --config: Set the location of a custom config file similar to `default.cfg` in the project root folder.
 - -v / --verbose: Display more in depth messages within the progress of CAPICE.
+- -vv / --debug: Display more in depth and debug messages within the progress of CAPICE.
 - -f / --force: Overwrite an output file if already present (does NOT work for logfiles).
 - --train: Activates the 'train new CAPICE-like models' within CAPICE.
```

Comment:
Proposal for the CLI:
```
capice.py --help
capice.py <command> --help
capice.py <command> <args>
```
resulting in e.g.:
```
capice.py predict <args>
capice.py train <args>
capice.py explain <args>
```
this could for example be implemented by having `capice.py` switch on the command and call `capice_<command>.py` with `args`.